index.bs

```diff
@@ -398,6 +398,17 @@ spec:url; type:dfn; text:scheme
     1. Let |resource| be a new [=request=] whose [=request URL|URL=] is |url| and whose [=request referrer policy|referrer policy=] is "{{no-referrer}}".
 
     1. [=Navigate=] the [=guest browsing context=] to |resource|.
+
+    <div class="note">
+      Unlike an <{iframe}> element, a <{portal}> element supports a state where
+      it has no associated browsing context. This is the initial state of a
+      <{portal}> element (i.e., it has no initial `about:blank` document, but
+      navigates directly to the first legal URL assigned to it).
```

Comment:
Nit: `document, but navigates` -> `document; instead it navigates`

Also maybe legal -> parseable

Reply:
Done, and changed to "parsable", which Wiktionary tells me is preferred for formal writing. (And what are we if not formal?)